wouso/interface/cpanel/forms.py

```diff
@@ -207,7 +207,35 @@ class Meta:
             'password': forms.PasswordInput(),
         }
         fields = ['username', 'first_name', 'last_name', 'email',
-                  'password', 'confirm_password', 'is_active']
+                'password', 'confirm_password', 'is_active']
+
+        def clean(self):
+            password1 = self.cleaned_data.get('password')
+            password2 = self.cleaned_data.get('confirm_password')
+            if password1 != password2:
+                raise forms.ValidationError("Passwords don't match")
+            return self.cleaned_data
+
+        def save(self):
+            self.instance.set_password(self.cleaned_data.get('password'))
+            self.instance.save()
+            return self.instance
+
+
+class UserForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ['username', 'first_name', 'last_name', 'email',
+                'is_active']
+
+
+class ChangePasswordForm(forms.ModelForm):
+    confirm_password = forms.CharField(widget=forms.PasswordInput())
+    password = forms.CharField(widget=forms.PasswordInput())
```

Comment:
```password``` is not required here. The form is ```ModelForm``` which means a form field will be created for each field of the model (```model = User``` in ```class Meta```).